architecture/ksm-metrics.md

```diff
@@ -132,3 +120,55 @@ Kubecost itself is resilient to duplicate metrics, but other services or queries
     * [Pod metrics](https://github.com/kubecost/cost-model/blob/0a0793ec040013fe44c058ff37f032449a2f1191/pkg/metrics/podlabelmetrics.go#L51-L60)
       * `kube_pod_labels`
       * `kube_pod_owner`
+
+## Adding external KSM metrics to Kubecost
```

Comment:
Can we provide some copy on the use cases that may lead to a user wanting to add their own KSM deployment?

Reply:
added to top.